packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/Wizard.js

```diff
@@ -92,10 +95,17 @@ function Wizard({
 }
 
 Wizard.propTypes = {
-  steps: PropTypes.array.isRequired,
-  initialValues: PropTypes.object,
+  stepReducer: PropTypes.func.isRequired,
+  initialSteps: PropTypes.arrayOf(
+    PropTypes.shape({
+      component: PropTypes.func.isRequired,
+      name: PropTypes.string.isRequired,
+      label: PropTypes.string.isRequired
+    })
+  ).isRequired,
   initialStepIndex: PropTypes.number,
-  onSubmit: PropTypes.func.isRequired
+  initialValues: PropTypes.object,
+  Rules: PropTypes.func
```

Comment:
`PropTypes.node` should it works for func and element)

Reply:
It seems to dislike it. Maybe it is because it is a basic function component ?
<img width="472" alt="Screenshot 2019-07-10 at 15 48 45" src="https://user-images.githubusercontent.com/705453/60974401-52865500-a32a-11e9-97d2-61556c12461c.png">